src/main/kotlin/com/github/mitchellolsthoorn/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.mitchellolsthoorn.testgenie.services
+
+import com.intellij.ide.highlighter.JavaFileType
+import com.intellij.openapi.project.Project
+import com.intellij.psi.JavaCodeFragmentFactory
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.ui.EditorTextField
+import com.intellij.ui.components.JBScrollPane
+import java.awt.BorderLayout
+import java.awt.Dimension
+import javax.swing.*
+
+class TestCaseDisplayService(private val project: Project) {
+
+    val mainPanel: JPanel = JPanel()
+    private val applyButton: JButton = JButton("Apply")
```

Comment:
Would buttons such as "select all" or "deselect all" be a part of this issue and pull request or not? Should we consider adding them?

Reply:
Those could indeed be useful, I agree. However, as this PR is blocking other work, maybe those buttons should be part of another PR.